moveit_planners/stomp/stomp_moveit/include/stomp_moveit/cost_functions/tool_goal_pose.h

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file tool_goal_pose.h
+ * @brief This defines a cost function for tool goal pose.
+ *
+ * @author Jorge Nicho
+ * @date June 2, 2016
+ * @version TODO
+ * @bug No known bugs
+ *
+ * @copyright Copyright (c) 2016, Southwest Research Institute
+ *
+ * @license Software License Agreement (Apache License)\n
+ * \n
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at\n
+ * \n
+ * http://www.apache.org/licenses/LICENSE-2.0\n
+ * \n
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef INDUSTRIAL_MOVEIT_STOMP_MOVEIT_INCLUDE_STOMP_MOVEIT_COST_FUNCTIONS_TOOL_GOAL_POSE_H_
```

Comment:
``INDUSTRIAL_MOVEIT_`` should likely change to ``MOVEIT_PLANNERS_STEMP_`` throughout